doc: add mention about dredd image (#172)
apiaryio/docker-base-images: README.md

Images are built automatically on Wercker. Builds on `master` branch include a deploy step with pushing the built
images to DockerHub. If you add a new image as described above, your image will get built and pushed automatically once the changes are merged.

## Dredd docker image

> Dredd is an HTTP API testing tool. You can find out more about it at [its documentation](https://github.com/apiaryio/dredd) or its [code repository](https://dredd.rtfd.io). Docker image [apiaryio/dredd](https://hub.docker.com/r/apiaryio/dredd/) are moved into [separate repository](https://github.com/apiaryio/dredd-docker).

Reviewer: ```suggestion
Dredd is an HTTP API testing tool. You can find out more about it at [its documentation](https://dredd.org) or its [code repository](https://github.com/apiaryio/dredd). Its Docker image [apiaryio/dredd](https://hub.docker.com/r/apiaryio/dredd/) has been moved to a [separate repository](https://github.com/apiaryio/dredd-docker).
```
